omero/users/whatsnew.txt

```diff
@@ -1,14 +1,21 @@
 What's new for OMERO 5.1
 ========================
 
-From a user's perspective, OMERO 5.1 introduces support for units and new more
-flexible user-added metadata. Other updates and new features include:
+Updates and new features for OMERO 5.1 include:
 
+- support for units
```

Comment:
Could we do something catchier here by appending "....35 fields throughout the model like ... can now be stored in arbitrary units (e.g. micrometers)"